test/acceptance/create_webdriver.robot

```diff
@@ -31,6 +31,39 @@ Create Webdriver With Bad Keyword Argument Dictionary
     Run Keyword And Expect Error    kwargs must be a dictionary.
     ...    Create Webdriver    Firefox    kwargs={'spam': 'eggs'}
 
+Get Browser Capabilities Using Dictionary
```

Comment:
Could you change the test that one test tests only a one thing. Also get rid of the stuff which is not anymore valid like the [Setup].